Utils/Accessibility.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+public protocol AccessibilityActionsSource: class {
+    func accessibilityCustomActionsForView(view: UIView) -> [UIAccessibilityCustomAction]?
+}
+
+public class Action {
```

Comment:
How about `AccessibleAction` (note the subtle suggestion of Accessib**le** vs Accessib**ility**)? That is, rather than saying that the action is *for* accessibility, we're saying the action is *compatible with* accessibility. Would we ever want to create actions that aren't accessible?

The reason we're against simply using `Action` is that Swift is namespaced per-target, meaning `Action` becomes globally visible to all classes in that target (which is the majority of the project). `Action` might mean something else entirely in a different context (especially a non-UI one), so the class name should be as specific as possible to avoid ambiguity. At the very least, I'd change `Action` -> `UIAction` if `AccessibleAction` isn't suitable.